core/tests/test_utils.py

```diff
@@ -466,6 +466,24 @@ def post_json(self, url, payload, csrf_token=None, expect_errors=False,
         return self._parse_json_response(
             json_response, expect_errors=expect_errors)
 
+    def delete_json(self, url, params='', expect_errors=False,
+                    expected_status_int=200):
+        """Delete object on the server by JSON; return the received object."""
```

Comment:
Drop the second clause, otherwise there's potential for confusion between the two uses of "object" (which actually mean different things).

Also, some minor rephrasing -- perhaps: "Delete object on the server using a JSON call."

Reply:
fixed.